circle.yml

```diff
@@ -0,0 +1,29 @@
+machine:
+  php:
+    # https://circleci.com/docs/environment#php
+    version: 5.6.22
+  environment:
+    # DB config. Using default CircleCI's database.
+    TERMINUS_ENV: ci-$CIRCLE_BUILD_NUM
+    TERMINUS_SITE: wp-redis
+
+dependencies:
+  cache_directories:
+    - ~/.composer/cache
+  pre:
+    # Set the PHP timezone so that Behat script does not fail.
```

Comment:
I just made the same change for saml https://github.com/pantheon-systems/wp-saml-auth/pull/21/commits/591ab44ee5a276fb0e2b23294a586a0bec4a5fe9